azblob/zt_sas_blob_snapshot_test.go

```diff
@@ -0,0 +1,42 @@
+package azblob_test
+
+import(
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	chk "gopkg.in/check.v1"
+	"os"
+	"time"
+)
+
+func (s *aztestsSuite) TestNewSASQueryParametersSnapshot(c *chk.C) {
```

Comment:
Please exercise the SAS, make sure it's signed correctly. For example:

1. Create a blob + a snapshot
2. Generate a snapshot SAS and read the snapshot, make sure the snapshot SAS cannot ready the blob itself